packages/elements/src/react/hooks/use-password.hook.ts

```diff
@@ -91,48 +91,48 @@ export const usePassword = (callbacks?: UsePasswordCallbacks) => {
   };
 };
 
-const get = (t: Record<string, any>, path: string) => path.split('.').reduce((r, k) => r?.[k], t);
+// const get = (t: Record<string, any>, path: string) => path.split('.').reduce((r, k) => r?.[k], t);
```

Comment:
What's up with that change?

Reply:
Unused code. I can just remove it instead